packages/server/src/bandit/banditSelection.ts

```diff
@@ -0,0 +1,75 @@
+import { EpicTest, EpicVariant } from '@sdc/shared/dist/types';
+import { BanditData } from './banditData';
+import { Result } from '../tests/epics/epicSelection';
+import { putMetric } from '../utils/cloudwatch';
+import { logError } from '../utils/logging';
+
+/**
+ * Select at random with probability epsilon.
+ * https://en.wikipedia.org/wiki/Multi-armed_bandit#Semi-uniform_strategies
+ */
+const EPSILON = 0.1;
```

Comment:
Can we change this to a meaningful name and reference that it's the epsilon value? Or alternatively add a description above. If you don't understand MABs you may not want to have to trawl through wikipedia to try and work it out.